misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -159,13 +160,15 @@ class HibernateModule(
   }
 
   private fun maybeBindStartVitessService() {
+    if (config.type != DataSourceType.VITESS) return
+
     val environment = Environment.fromEnvironmentVariable()
-    if (environment == Environment.DEVELOPMENT || environment == Environment.TESTING) {
-      val startVitessServiceKey = StartVitessService::class.toKey(qualifier)
-      multibind<Service>().to(startVitessServiceKey)
-      bind(startVitessServiceKey).toProvider(Provider<StartVitessService> {
-        StartVitessService(environment = environment, config = config, qualifier = qualifier)
-      }).asSingleton()
-    }
+    if (environment !in setOf(Environment.DEVELOPMENT, Environment.TESTING)) return
```

Comment:
/nit mind adding some brackets on these one-liners? I find them rather confusing and am forever scared of https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/

Reply:
good story! will change